impl/MetadataApiImpl.h

```diff
@@ -0,0 +1,43 @@
+/*
+* MetadataApiImpl.h
+*
+*
+*/
```

Comment:
Right, I've removed those headers because it seemed faster to just copy and paste the already existing files to use as a template than to learn Swagger. Maybe it's super easy, but I haven't felt inconvenienced by not using it yet.

Reply:
Some things were generated, others were not.

We are going to refactor this project in the future so let's let it ride for now. As long as things follow the same patterns, we will be okay.